src/js/common/schemaform/reducers/index.js

```diff
@@ -144,14 +144,14 @@ export default function createSchemaFormReducer(formConfig) {
         return _.set('savedStatus', action.status, state);
       }
       case SET_FETCH_FORM_STATUS: {
-        const newState = _.set('loadedStatus', action.status, state);
+        return _.set('loadedStatus', action.status, state);
+      }
+      case SET_IN_PROGRESS_FORM: {
+        const newState = _.set('loadedData', action.data, state);
         newState.loadedStatus = LOAD_STATUSES.success;
```

Comment:
Realized I put this in the wrong `case`. Now it'll only set the load status to `success` after we successfully load, which, y'know, makes sense.